portal-frontend/src/app/services/authentication/authentication.service.ts

```diff
@@ -79,13 +79,10 @@ export class AuthenticationService {
 
   async refreshTokens(redirect = true) {
     if (this.refreshToken) {
-      if (this.expires && this.expires < Date.now()) {
-        if (redirect) {
-          await this.router.navigateByUrl('/login');
-        }
+      if (this.refreshExpires && this.refreshExpires < Date.now()) {
+        await this.router.navigateByUrl('/login');
```

Comment:
Did you mean to remove the if(redirect) check?

Reply:
added it back